tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -504,6 +507,8 @@ def run_test(self, test):
 
         stack = list(((test, item[0]), item[1]) for item in reversed(test.references))
         page_idx = None
```

Comment:
So we depend on this being set in the loop below and if it's ever `None` that's now an error in the code. So I think we should remove the declaration which sets it to `None`.

Reply:
done